__tests__/helpers/database.js

```diff
@@ -0,0 +1,36 @@
+import {
+  internet, lorem, name,
+} from 'faker';
+import _ from 'lodash';
+
+export default (app) => ({
+  insert: {
+    status: (data = {}) => app.objection.models.status.query()
+      .insert(_.merge({ name: name.title() }, data)),
```

Comment:
i'd offer extract data building from here. it could be useful when we need to do insert.status(data)

Reply:
and there is no reason to prefer _.merge over object spreading

Reply:
not sure about first point,
about second - right now no, because all fields are flat...do you want me to replace that?

Reply:
if fields are not flat better to use defaultsDeep

Reply:
so it will be like `defaults(data, {...})` ?

Reply:
_.defaults mutates data, so better _.defaults({...data}, {...})